python/requirements_test.txt

```diff
@@ -31,26 +31,26 @@ gradio==3.5; platform_system != "Windows"
 jsonpatch==1.32
 kubernetes==24.2.0
 # higher version of llvmlite breaks windows
-llvmlite==0.39.1; platform_system != "Windows"
-llvmlite==0.34.0; platform_system == "Windows"
+llvmlite==0.39.1; platform_system != "Windows" and python_version < '3.11'
+llvmlite==0.34.0; platform_system == "Windows" and python_version < '3.11'
 lxml==4.9.1
 moto[s3,server]==4.0.7
 mypy==0.982
 networkx==2.6.3
-numba==0.56.3; platform_system != "Windows"
-numba==0.51.2; platform_system == "Windows"
+numba==0.56.3; platform_system != "Windows" and python_version < '3.11'
+numba==0.51.2; platform_system == "Windows" and python_version < '3.11'
 openpyxl==3.0.10
 opentelemetry-api==1.1.0
 opentelemetry-sdk==1.1.0
 opentelemetry-exporter-otlp==1.1.0
 pexpect==4.8.0
 Pillow==9.2.0; platform_system != "Windows"
 proxy.py==2.4.3
-pyarrow==6.0.1
-pydantic==1.9.2
```

Comment:
should we do conditional on the python version as in `python/requirements.txt`? 
Could this be why the test `tests:test_pydantic_serialization` was failing?

Reply:
Let's still do the conditional as other dependencies for 3.11.

Reply:
I forgot what the error message was, but I believe we need a newer pydantic for 3.11. I haven't figured out why that test fails.

Reply:
Looks like it's working now?